src/actions/channels.js

```diff
@@ -181,6 +181,15 @@ export function createGroupChannel(userIds: Array<string>): ActionFunc {
             last_update_at: created.create_at,
         };
 
+        if (created.create_at !== created.update_at) {
+            try {
+                member = await Client4.getMyChannelMember(created.id);
```

Comment:
Why not use the `getMyChannelMember` action instead of directly calling the client?

Reply:
I going to add a first try to get my member from the store, but it is very possible that you don't have it in the store, because you are opening a new channel, so, probably is because your previous group chat was hidden, so, probably you don't have it in the store.

Reply:
Done